EnglishFilter/sections/specific.txt

```diff
@@ -254,6 +254,7 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 !
+earncoin.site##.content
```

Comment:
I think we should not hide this element, btw this rule is dangerous

![Screenshot 2021-02-17 at 15 07 06](https://user-images.githubusercontent.com/18536328/108202295-f30d3f80-7131-11eb-8231-118f5d25c5df.png)

Reply:
@Sergey-Lyapin That's crap, would the person read that while waiting?

Reply:
> That's crap, would the person read that while waiting?

Not an ad.

Reply:
Ok removed.